ci/terraform/modules/endpoint-lambda/alerts.tf

```diff
@@ -17,6 +17,13 @@ resource "aws_cloudwatch_log_metric_filter" "lambda_error_metric_filter" {
   }
 }
 
+locals {
+  base_error_alarm_description      = "${var.lambda_log_alarm_threshold} or more errors have occurred in the ${var.environment} ${var.endpoint_name} lambda.ACCOUNT: ${var.account_alias}"
+  error_alarm_description           = var.runbook_link == null ? local.base_error_alarm_description : "${local.base_error_alarm_description}. Runbook: ${var.runbook_link}"
+  base_error_rate_alarm_description = "Lambda error rate of ${var.lambda_log_alarm_error_rate_threshold} has been reached in the ${var.environment} ${var.endpoint_name} lambda.ACCOUNT: ${var.account_alias}"
+  error_rate_alarm_description      = var.runbook_link == null ? local.base_error_rate_alarm_description : "${local.base_error_rate_alarm_description}. Runbook: ${var.runbook_link}"
+}
+
```

Comment:
These Locals are already defined  on line no 1 Throwing exception

│ A local value named "base_error_alarm_description" was already defined at
│ ../modules/endpoint-lambda/alerts.tf:2,3-187. Local value names must be
│ unique within a module.

Reply:
oops. fixed